core/src/main/java/com/scalar/db/api/GetBuilder.java

```diff
@@ -79,6 +80,11 @@ public Buildable projections(Collection<String> projections) {
       return this;
     }
 
+    @Override
+    public Buildable projections(String... projections) {
+      return projections(Arrays.asList(projections));
+    }
```

Comment:
In the `Get.BuildableFromExisting` class, I think it is necessary to override this method to make it return the `this` object of type `Get.BuildableFromExisting`. Otherwise, while calling this method from the `Get.BuildableFromExisting` object, it will return a `Get.Buildable` object.

Reply:
Fixed in the latest commit.